controllers/cindervolume_controller.go

```diff
@@ -774,22 +783,38 @@ func processCustomServiceConfig(customServiceConfig string) (bool, string) {
 		} else if strings.HasPrefix(token, "[") && strings.HasSuffix(token, "]") {
 			// Note the section name before looking for a volume_backend_name
 			sectionName = strings.Trim(token, "[]")
+
+			if isBackendSection && isLVM {
```

Comment:
I could follow this because I wrote the original code, but it took me a while to remember the parsing rules. I think what would help here is the block from L787..L792 is processing the previous (if any) backend section. It may not be intuitive that you are basically post-processing the previous section immediately after identifying a NEW section on L783.